### PR TITLE
Reorg Vsync as a service. It's now necessary because it has deferred queues.

### LINKED ISSUE #880: Create vsyncFor and cleanup uses

### PR DESCRIPTION
**Pull request by dvoytenko**
Closes #880.

### CODE REVIEW

File: src/swipe.js [hunk 1]
```diff
@@ -1,537 +0,0 @@
-/**
```
Comment (cramforce) [inline src/swipe.js]: Do you really want to delete this?

File: test/functional/test-swipe.js [hunk 2]
```diff
@@ -1,404 +0,0 @@
-/**
```
Comment (cramforce) [inline test/functional/test-swipe.js]: Delete this?

**Comment by cramforce** [comment]
LGTM but see questions

File: src/swipe.js (hunk 1 above)
Comment (dvoytenko) [inline src/swipe.js]: Yeah. It was deprecated for a long time. No longer used. Easier to delete now than to refactor.

File: test/functional/test-swipe.js (hunk 2 above)
Comment (dvoytenko) [inline test/functional/test-swipe.js]: Yeah. It was deprecated for a long time. No longer used. Easier to delete now than to refactor.

**Pull request merged**